src/scenes/Users/Detail/UserDetail.tsx

```diff
@@ -111,6 +115,24 @@ export const UserDetail = () => {
             loading={!user}
           />
           {user ? <EditUser user={user} {...editUserState} /> : null}
+
+          <DisplayProperty
+            label="Partners"
+            value={
+              user?.organizations.items.length ? (
+                <Box mt={1}>
```

Comment:
Let's not use Box outside of StoryBook. It creates a new css class every time it's mounted.

Reply:
Got it, so just use div with `className` prop?

Reply:
Yeah that would be great :)

Reply:
Just updated to div+class instead of Box